src/main/java/org/alfresco/mockeventgenerator/config/amqp/AmqpProperties.java

```diff
@@ -18,22 +18,24 @@
 import org.alfresco.mockeventgenerator.config.CamelRouteProperties;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * @author Jamal Kaabi-Mofrad
  */
 @ConfigurationProperties(prefix = "messaging.to.activemq")
 public class AmqpProperties
 {
-    private final CamelRouteProperties camelRoute = new CamelRouteProperties();
+    private final List<CamelRouteProperties> camelRoutes = new ArrayList<>();
     private String host;
     private int port;
     private String username;
     private String password;
     private String url;
 
-    public CamelRouteProperties getCamelRoute()
-    {
-        return camelRoute;
+    public List<CamelRouteProperties> getCamelRoutes() {
```

Comment:
Small format fix required here, with bracket on new line and remove the extra spaces at the end.